test/js/utils/mochaw.js

```diff
@@ -46,3 +46,15 @@ export const instantiate = (...classFiles) => (...args) =>
       [uncapitalize(classFile)]: instantiateFile(classFile)(...args)
     })
   ,{})
+
+export const locale = (lang="en") => require(`../../../src/locales/${lang}.json`)
+
+export const verifyElementCopy = async (element, copy) => {
+  try {
+    const elementText = await element.getText()
+    await expect(elementText).to.equal(copy)
+    await element.isDisplayed()
+  } catch (error) {
```

Comment:
if you don't catch the error it might actually fail the test

Reply:
After removing the catch, I got plenty of `UnhandledPromiseRejectionWarning: StaleElementReferenceError: stale element reference: element is not attached to the page document` errors for each test though they didn't fail the tests.

Seems I can get rid of these warnings/errors by adding `async` to each `verify` function in page objects classes and then I get the failures of the tests.